SpatialGDK/Source/SpatialGDKServices/Public/SpatialGDKServicesConstants.h

```diff
@@ -61,4 +69,7 @@ const FString ProxyInfoFilePath = FPaths::Combine(ProxyFileDirectory, TEXT("Serv
 const FString LsofCmdFilePath = TEXT("/usr/sbin/");
 const FString KillCmdFilePath = TEXT("/bin/");
 #endif
+
+const int32 ExitCodeSuccess = 0;
```

Comment:
there are so many more places where we can use this <3

Reply:
Event tracing api exit code success is 1 :(

Reply:
That's so strange, 0 is globally exit code success, 1 is failure, thought that was for..... everything

Reply:
Yeah same, trolled me pretty hard.